test/e2e/common/node/framework/podresize/resize.go

```diff
@@ -202,6 +203,13 @@ func verifyPodContainersStatusResources(gotCtrStatuses []v1.ContainerStatus, wan
 			errs = append(errs, fmt.Errorf("container status %d name %q != expected name %q", i, gotCtrStatus.Name, wantCtr.Name))
 			continue
 		}
+
+		// Convert 'got' memory resources to the same format as 'want' memory resources for struct comparison.
```

Comment:
So I ran into [this error again](https://github.com/kubernetes/kubernetes/pull/132152#discussion_r2198569071) with the local run and decided to dig into it this morning. I think it was just that the underlying struct for the resource quantities had slightly different formats for memory for whatever reason. These lines I added normalizes the format without changing the value so that the structs can be compared properly.